docker-compose.yml

```diff
@@ -11,12 +11,13 @@ services:
     environment:
       KAFKA_ADVERTISED_HOST_NAME: ${DOCKER_HOST_IP}
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_CREATE_TOPICS: "topic.EncodedBytes:1:1,topic.DecodedJson:1:1"
+      KAFKA_CREATE_TOPICS: "j2735asn1xer:1:1,j2735asn1per:1:1"
```

Comment:
These topic names are not very relevant. ODE uses different names for the input and output topics but that's not important since this module is a general purpose module and not necessarily for ODE. That said, j2735asn1xer and j2735asn1per are only applicable to the encoding function. For decoding, the module receives XML (not XER) that contains hex bytes and outputs XER. Additionally, they will not always be J2735. I think these topic names should be made more generic, such as `acminput` and `acmoutput`.